packages/api/src/user.js

```diff
@@ -700,13 +700,15 @@ export async function userPaymentPut (request, env) {
       billing: env.billing,
       customers: env.customers,
       user: { ...request.auth.user, id: request.auth.user._id },
-      subscriptions: env.subscriptions
+      subscriptions: env.subscriptions,
+      termsOfService: env.termsOfService
```

Comment:
In the long run, what do you think of using this service and table for 'agreements' or 'consents' beyond just terms of service agreements? It might be nice to model things that way. Curious for your thoughts, we can always iterate towards that after this PR if it makes sense to all.